ops/Dockerfile-consensus

```diff
@@ -19,8 +19,9 @@ RUN apt-get update -q -q && \
 
 # Install SGX Ubuntu/Debian Repo
 RUN source /etc/os-release && \
-	wget "https://download.01.org/intel-sgx/sgx-linux/2.15/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_driver_2.11.0_2d2b795.bin" && \
-	wget "https://download.01.org/intel-sgx/sgx-linux/2.15/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_sdk_2.15.100.3.bin" && \
+	wget "https://download.01.org/intel-sgx/sgx-linux/2.16/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_driver_2.11.054c9c4c.bin" && \
+
+	wget "https://download.01.org/intel-sgx/sgx-linux/2.16/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_sdk_2.16.100.4.bin" && \
```

Comment:
That's a good question, looks like the lines got added as part of 600812bf1aa2b3af8163ecd814877ff75991750e, however not seeing anything explicitly utilizing the binaries, even back then.  @jcape Do you know why these `wget`s are here?

Reply:
I don't know off the top of my head, this would have been added in a release branch, probably just cargo-culted from somewhere else. I don't think it hurts anything to leave this here for now, we can/should do a larger cleanup of our dockerfiles later.